SlideServer.py

```diff
@@ -223,8 +223,10 @@ def singleThumb(filepath):
 
 @app.route("/data/many/<filepathlist>", methods=['GET'])
 def multiSlide(filepathlist):
-    request.args.get('extended')
-    res = dev_utils.getMetadataList(json.loads(filepathlist), app.config['UPLOAD_FOLDER'], extended)
+    extended = request.args.get('extended')
+    filenames = json.loads(filepathlist)
+    paths = [join(app.config['UPLOAD_FOLDER'], filename) for filename in filenames]
```

Comment:
Optinal suggestion for readability, since we're using str's join in these files too.

```suggestion
    paths = [os.path.join(app.config['UPLOAD_FOLDER'], filename) for filename in filenames]
```

Reply:
@birm Good idea, done